wix/Includes/OpenJDK.Variables.wxi.template

```diff
@@ -13,7 +13,7 @@
 
   <?define ProductVersion="$(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion)" ?>
   <?define ProductLanguage="!(loc.ProductLanguage)" ?>
-  <?define ProductManufacturer="AdoptOpenJDK" ?>
+  <?define ProductManufacturer="{vendor}OpenJDK" ?>
```

Comment:
Vendor variable should replace the entire string. "DragonwellOpenJDK" or "CorrettoOpenJDK" would be both wrong and bad.